nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/gridfs/PutGridFS.java

```diff
@@ -243,6 +299,14 @@ private Document getMetadata(FlowFile input, ProcessContext context) {
             }
         }
 
+        String hashAttribute = context.getProperty(HASH_ATTRIBUTE).evaluateAttributeExpressions().getValue();
+        String hashMetadata  = context.getProperty(HASH_METADATA_FIELD_NAME).evaluateAttributeExpressions().getValue();
+        String hashValue     = input.getAttribute(hashAttribute);
+
+        if (!StringUtils.isEmpty(hashValue)) {
```

Comment:
This could use `StringUtils.isNotEmpty()`

Reply:
Done